src/ConfigProvider.php

```diff
@@ -11,6 +11,7 @@
 
 use Psr\Http\Message\ServerRequestInterface;
 use Zend\HttpHandlerRunner\RequestHandlerRunner;
+use function extension_loaded;
```

Comment:
?

Reply:
Why not?

Reply:
@snapshotpl check the whole file, the function is used in line 20.